database/index.js

```diff
@@ -14,4 +14,33 @@ connection.connect((err) => {
   console.log('connected');
```

Comment:
Toby suggested this for me, so I'll pass it onto you. You should separate the db connection from db queries into 2 different files.